spec/features/admin/admin_creates_new_admin_account_spec.rb

```diff
@@ -2,20 +2,18 @@
 
 feature "admin creates new admin account" do
   scenario "successfully" do
-    skip
-    user = create(:user, admin: true)
+    user = create(:user, :admin)
     log_in_as user
 
-    visit new_user_path
-
+    visit new_admin_user_path
     fill_in "user_name", with: "Mary"
     fill_in "user_email", with: "Mary@example.com"
-    check "user_admin"
+    choose "Yes"
     fill_in "user_password", with: "secret"
-    click_on I18n.t("helpers.submit.user.create")
+    click_on I18n.t("admin.users.new.submit")
 
-    expect(user.admin).to eq true
-    expect(page).to have_text(I18n.t("users.show.welcome"))
+    expect(user).to be_admin
```

Comment:
This assertion is redundant, since this user was created by us in a factory. What we want to see is that we created a new admin called Mary. So a suggestion to change this to be:
```
user = User.last

within "#user_#{user.id}" do
  expect(page).to have_text("Mary")
  expect(page).to have_text("true")
end
```

Reply:
Duh, yep, thanks for pick up and the tests - done in sha: db0a541